src/io/flutter/ProjectOpenActivity.java

```diff
@@ -50,10 +50,17 @@ public void runActivity(@NotNull Project project) {
     if (FlutterUtils.isAndroidStudio() && AndroidUtils.isAndroidProject(project)) {
       AndroidUtils.addGradleListeners(project);
     }
-    if (!FlutterModuleUtils.declaresFlutter(project)) {
+
+    if (!FlutterModuleUtils.declaresFlutter(project) && !WorkspaceCache.getInstance(project).isBazel()) {
```

Comment:
The fact that 
FlutterModuleUtils.declaresFlutter(project) returns false for Bazel projects is probably a bug. We should fix that instead of working around it here. Audit all the other uses to make sure none will be problems for bazel.